geometry_msgs/Pose diff
#NodeMsg prev #Those pointers in the code so retrieve ids
#NodeMsg next #Those pointers in the code so retrieve ids

std_msgs/UInt64 prev
std_msgs/UInt64 next

std_msgs/Float64MultiArray covariance